src/EditorFeatures/VisualBasic/LineCommit/CommitBufferManager.vb

```diff
@@ -156,8 +168,10 @@ Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.LineCommit
         End Sub
 
         Private Structure FormattingInfo
-            Public UseSemantics As Boolean
-            Public SpanToFormat As SnapshotSpan
+            Public _useSemantics As Boolean
+            Public _useStatementSemanticFormatting As Boolean
+            Public _spanToFormat As SnapshotSpan
+            Public _statementSpanToSemanticFormat As SnapshotSpan
```

Comment:
please use hte original naming style here.

Reply:
please doc what all of these mean.  thanks!

Reply:
Fixed